src/controllers/booleansControllers.js

```diff
@@ -0,0 +1,41 @@
+const { negate, truthiness, isOdd, startsWith } = require('./../lib/booleans');
+
+exports.negate = (req, res) => {
+  const { value } = req.body;
+
+  res.status(200).json({
+    result: negate(value),
+  });
+};
+
+exports.truthiness = (req, res) => {
+  const truth = req.body.value;
+
+  res.status(200).json({
+    result: truthiness(truth),
+  });
+};
+
+exports.isOdd = (req, res) => {
+  const number = parseInt(req.params.num);
+
+  return Number.isNaN(number)
+    ? res.status(400).json({
```

Comment:
Nothing wrong with what you've written here. If it makes sense to you and you think it would make sense to other people looking at your code for the first time and trying to work it out then that's the best you can hope for :)

Usually I try to keep code as small as possible within ternaries, in order to make them as easy as possible to read. It could be clearer to have an object and a status that changes if Number.isNaN(number) is true or false. Eg
let status = 200;
let responseObject = { 
       result: isOdd(number),
};
if (Number.isNaN(number)) {
     status = 400;
     responseObject = {
            error: 'Parameter must be a number.',
      }
}
res.status(status).json(responseObject)